packages/react/src/components/builder-content.component.tsx

```diff
@@ -90,6 +90,7 @@ export class BuilderContent<ContentType extends object = any> extends React.Comp
     };
     if (this.props.content && !options.initialContent?.length) {
       options.initialContent = [this.props.content];
+      options.key = this.props.content.id;
```

Comment:
Two possible concerns here:
1. What if there is `initialContent`, don't we still want to set `options.key` to fix for that case with two builder components on the page? It feels like maybe we don't need to tie setting `options.key` to `!options.initialContent?.length` like we do here
2. What if `options` already has a `key` set? We will override it here, which does not seem like the right behavior.

What if instead we added an additional condition like this:
```typescript
get options() {
  let options = {
    ...(this.props.options || ({} as GetContentOptions)),
  };
  if (!options.key && this.props.content?.id) {
    options.key = this.props.content.id;
  }
.....
    
```

Reply:
Thanks @mrkoreye for your comment. I have updated the PR with the checks.